src/common/__tests__/data/generics.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ *  Various primitive/generic values for use in testing.
+ *  Because hard-coding constants is unreliable and annoying.
+ */
+
+/**
+ *  An array of strings
+ */
+
+export const array: string[] = ['Life', 'Universe', 'Everything'];
+
+/**
+ *  A single string
+ */
+
+export const string = "Don't Panic";
+
+/**
+ *  A single string with no spaces or special characters
+ */
+
+export const safeString = 'dont-panic';
+
+/**
+ *  An integer
+ */
+
+export const int = 42;
+
+/**
+ *  A single number representing a year
+ */
+
+export const year = 2018;
```

Comment:
Is there any mileage in using the current year here? Or would you prefer to keep this fairly static?

```suggestion
export const year: string = new Date().getFullYear();
```

Reply:
tbh, it doesn't look like we even ended up using the dummy year value in the eaf project. My inclination is that the value in our test data should be static and predictable, since we can always create a more specific `currentYear` value if needed.

Reply:
That seems reasonable, yeah.